apps/server/src/modules/task/service/task.service.spec.ts

```diff
@@ -198,6 +198,29 @@ describe('TaskService', () => {
 
 				lessonRepo.findById.mockRestore();
 			});
+			it('should throw if not all users do not belong to course', async () => {
+				course = courseFactory.studentsWithId(2).buildWithId();
+				const someUser = userFactory.buildWithId();
+
+				await expect(async () => {
+					await taskService.create(user.id, { name: 'test', courseId: course.id, usersIds: [someUser.id] });
+				}).rejects.toThrow(BadRequestException);
+			});
+			/*
+			it('should set users', async () => {
```

Comment:
what about this commented test?

Reply:
this test simply would not work, even if we have coverage because of api tests, I wanted to keep it so we can discuss it